### PR TITLE
Credentials look like Jira2FA tokens

### PR DESCRIPTION
**Pull request by babenek**
3 credentials found look like Jira2FA tokens 
for https://github.com/Samsung/CredSweeper/pull/311

### CODE REVIEW

**Review by csh519 — COMMENTED** [review]
I think those values are `Password`.
Please check.

File: meta/e3377359.csv [hunk 1]
```diff
@@ -116,4 +116,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 130300,1fd373f0,GitHub,e3377359,data/e3377359/src/v.php,420:420,F,F,,,F,F,,,,,0.00,,F,F,F,
 130301,1fd373f0,GitHub,e3377359,data/e3377359/src/v.php,408:408,F,F,,,F,F,,,,,0.00,,F,F,F,
 131695,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,110:110,T,F,,,F,F,,,,,0,0,F,F,F,Other
+131702,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,38:38,T,F,35,51,F,F,,,,,0,0,F,F,F,Other
+131703,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,74:74,T,F,35,51,F,F,,,,,0,0,F,F,F,Other
+131704,87fccb9b,GitHub,e3377359,data/e3377359/src/r.php,355:355,T,F,169,185,F,F,Any,,,,0,0,F,F,F,Other
```
Comment (babenek) [inline meta/e3377359.csv:121]:
```suggestion
131702,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,38:38,T,F,35,51,F,F,,,,,0,0,F,F,F,Password
131703,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,74:74,T,F,35,51,F,F,,,,,0,0,F,F,F,Password
131704,87fccb9b,GitHub,e3377359,data/e3377359/src/r.php,355:355,T,F,169,185,F,F,Any,,,,0,0,F,F,F,Password
```

**Review by csh519 — APPROVED** [review]
Thank you for adding new data.

LGTM 👍

**Review by iuriimet — APPROVED** [review]

**Review by kmnls — COMMENTED** [review]
Agree

**Pull request merged**